constraints/alignWrap.js

```diff
@@ -65,7 +108,10 @@ function alignLeftPS(sTree, pTree, cat){
 function alignRightPS(sTree, pTree, cat){
 	return alignPS(sTree, pTree, cat, 'right');
 }
-
+function alignCustom(sTree, pTree, cat, options){
+	options = options || {};
+	return alignSP()
```

Comment:
Shouldn't this return alignPS(sTree, pTree, cat, options)? This looks like it will always return 0 since no arguments are passed